catalog/transformer/catalog-transformer-pptx/pom.xml

```diff
@@ -0,0 +1,253 @@
+                        <Import-Package>
+                            !com.ibm.icu.text,
+                            !org.etsi.uri.x01903.v14.*,
+                            ddf.catalog.*,
+                            org.slf4j.*,
+                            com.google.common.io.*,
+                            org.xml.sax.*,
+                            org.w3c.dom.*,
+                            javax.xml.*,
+                            javax.imageio.*,
+                            org.apache.commons.logging.*
+                        </Import-Package>
+                        <Embed-Dependency>
+                            catalog-core-api-impl,
+                            platform-util,
+                            catalog-transformer-common,
+
+                            <!-- external -->
+
+                            bcprov-jdk15on,
+                            bcpkix-jdk15on,
+                            <!--
+
+                            TODO Big nasty hack. Should not normally embed another bundle. But we
+                            can't install this bundle separately in osgi because of a missing
+                            class in the bundle. So far, I've sent an email to
+                            users@servicemix.apache.org asking for input, and they created a
+                            ticket for it. When the next servicemix release is made, we should
+                            check to see if the error has been fixed, and update accordingly.
+
+                            This is the apache servicemix ticket:
+
+                            https://issues.apache.org/jira/browse/SM-2925
+
+                            See DDF Jira: https://codice.atlassian.net/browse/DDF-1986
+
+                            -->
+                            org.apache.servicemix.bundles.poi,
+
+                            commons-io,
+                            commons-lang3,
+                            fontbox,
+                            pdfbox,pdfbox-tools
+                        </Embed-Dependency>
+                        <Export-Package/>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>true</haltOnFailure>
+                            <rules>
+                                <rule>
+                                    <element>BUNDLE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>BRANCH</counter>
```

Comment:
You need to specify the limits for all four coverage ratios. Any limits you don't specify will be set to 0.